lib/ansible/playbook/play.py

```diff
@@ -64,6 +64,9 @@ class Play(Base, Taggable, Become):
 
     # Connection
     _gather_facts        = FieldAttribute(isa='bool', default=None, always_post_validate=True)
+    _gather_subset       = FieldAttribute(isa='string', default=None, always_post_validate=True)
```

Comment:
Doing this modification and using the following playbook rise an error:

```yaml
- hosts: localhost
  gather_subset: "min,network"
  tasks: []
```
The error raised:
```
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: TypeError: Bad subset 'min,network' given to Ansible. gather_subset options allowed: hardware, all, network, virtual, min
fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "parsed": false}
```